tests/e2e/utils/Logger.ts

```diff
@@ -29,15 +65,30 @@ export abstract class Logger {
      * @param text log text
      */
     public static trace(text: string) {
-        if (TestConstants.TS_SELENIUM_LOG_LEVEL === 'INFO') {
+        if (TestConstants.TS_SELENIUM_LOG_LEVEL === 'ERROR' ||
+            TestConstants.TS_SELENIUM_LOG_LEVEL === 'WARN' ||
+            TestConstants.TS_SELENIUM_LOG_LEVEL === 'INFO' ||
+            TestConstants.TS_SELENIUM_LOG_LEVEL === 'DEBUG') {
             return;
         }
 
-        if (TestConstants.TS_SELENIUM_LOG_LEVEL === 'DEBUG') {
-            return;
-        }
-
-        console.log(`            ‣ ${text}`);
+        this.logText(`            ‣ ${text}`);
     }
 
+    private static logText(text: string) {
```

Comment:
Wouldn't it be easier/nicer instead of counting whitespaces just put the desired indentation level as an argument? :-D

Reply:
Could also be possible. I just didn't want to make too many changes.

Reply:
fixed it now.